kind/README.md

```diff
@@ -0,0 +1,111 @@
+# Kind Deployment Guide
+
+This is a simplified deployment of Kuma and the Demo application on top of [kind](https://kind.sigs.k8s.io/). It follows the steps described in [kubernetes](../kubernetes/README.md) folder, replacing the Minikube deployment.
+
+## Prerequisites
+
+There are only two prerequisites required before you run the demo: [docker](https://docs.docker.com/get-docker/) and [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
```

Comment:
if im not mistaken, user also needs to have kind installed?

Reply:
Actually no. The script will download the fixed version of KIND that is specified in it. I have just updated it to the lates 0.8.0 released a couple days ago.